drivers/aks/aks_driver.go

```diff
@@ -92,134 +157,195 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 	driverFlag := types.DriverFlags{
 		Options: make(map[string]*types.Flag),
 	}
+
+	driverFlag.Options["subscription-id"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "Subscription credentials which uniquely identify Microsoft Azure subscription.",
+	}
+	driverFlag.Options["resource-group"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "The name of the 'Cluster' resource group.",
+	}
 	driverFlag.Options["name"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The internal name of the cluster in Rancher",
+		Usage: "The name of the 'Cluster' resource, and the internal name of the cluster in Rancher.",
 	}
-	driverFlag.Options["display-name"] = &types.Flag{
+
+	driverFlag.Options["aad-client-app-id"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The name that is displayed to the user on the Rancher UI",
+		Usage: `The ID of an Azure Active Directory client application of type "Native". This application is for user login via kubectl.`,
 	}
-	driverFlag.Options["subscription-id"] = &types.Flag{
+	driverFlag.Options["aad-server-app-id"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "Subscription credentials which uniquely identify Microsoft Azure subscription",
+		Usage: `The ID of an Azure Active Directory server application of type "Web app/API". This application represents the managed cluster's apiserver (Server application).`,
 	}
-	driverFlag.Options["resource-group"] = &types.Flag{
+	driverFlag.Options["aad-server-app-secret"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The name of the resource group",
+		Usage: `The secret of an Azure Active Directory server application.`,
 	}
-	driverFlag.Options["location"] = &types.Flag{
+	driverFlag.Options["aad-tenant-id"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "Resource location",
-		Default: &types.Default{
-			DefaultString: "eastus",
-		},
+		Usage: `The ID of an Azure Active Directory tenant.`,
 	}
-	driverFlag.Options["tags"] = &types.Flag{
-		Type:  types.StringSliceType,
-		Usage: "Resource tags. For example, foo=bar",
+
+	driverFlag.Options["enable-http-application-routing"] = &types.Flag{
+		Type:  types.BoolPointerType,
```

Comment:
Typically when we use bool pointers in KE it is because we are added a new value and it needs to default to false.  Should this be a regular bool or have a default value or `true`?

Reply:
Change to `BoolType` and use `True` as default.